processor/processor.go

```diff
@@ -183,6 +189,11 @@ func (proc *HandleT) Start() {
 	rruntime.Go(func() {
 		proc.mainLoop()
 	})
+	rruntime.Go(func() {
```

Comment:
Should this move out to main.go like how processor, router and batchrt are started controlled by a flag?